build/charts/theia/values.yaml

```diff
@@ -0,0 +1,27 @@
+clickhouse:
+  # -- Container image to use for the ClickHouse Monitor.
+  monitorImage:
+    repository: "projects.registry.vmware.com/antrea/theia-clickhouse-monitor"
+    pullPolicy: "IfNotPresent"
+    tag: "latest"
+  # -- ClickHouse username. It will be stored in a secret.
+  username: "clickhouse_operator"
+  # -- ClickHouse password. It will be stored in a secret.
+  password: "clickhouse_operator_password"
+  # -- HTTP port number for the ClickHouse service.
+  httpPort: 8123
+  # -- TCP port number for the ClickHouse service.
+  tcpPort: 9000
```

Comment:
does it sometimes make sense to disable one of the ports to only use HTTP or TCP?

Reply:
I think the Flow Aggregator uses TCP to connect the ClickHouse while Grafana uses HTTP for connection. Thus it might not have a use case to disable one of them.